library/src/main/java/com/google/android/exoplayer/text/webvtt/WebvttParser.java

```diff
@@ -46,6 +47,10 @@
   private static final Pattern WEBVTT_FILE_HEADER =
       Pattern.compile(WEBVTT_FILE_HEADER_STRING);
 
+  private static final String WEBVTT_COMMENT_BLOCK_STRING = "^NOTE((\\u0020|\u0009).*)?$";
```

Comment:
In fact, even without talking strictly, "^NOTE\s" is not valid. 

For example, given the following text block:

NOTE
This is a sample comment

As the string returned by readLine method of BufferedReader doesn't include the newline sequence, "^NOTE\s" wouldn't be correct to validate the previous example comment.